tensorflow_addons/image/tests/filters_test.py

```diff
@@ -359,3 +362,111 @@ def test_symmetric_padding_with_3x3_filter_median(image_shape):
         constant_values=0,
         expected_plane=expected_plane,
     )
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_gaussian_filter2d_constant():
+    test_image_tf = tf.random.uniform(
+        [1, 40, 40, 1], minval=0, maxval=255, dtype=tf.float64
```

Comment:
Why `float64`? IMO `float32` would be fine. Because everything will be in `float32`, it will be casted to float32 anyways in that case

Reply:
Actually in the exponentiation, I am using float64. So to keep the data type uniform over all the operations I am using double.

Reply:
Is `float32` causing overflow?

Reply:
Done